_sass/_toc.scss

```diff
@@ -0,0 +1,14 @@
+.toc {
+  background-color: #f6f7f9;
+  padding: 8px;
+}
+
+.toc {
+    ul, li {
```

Comment:
could maybe move the ul, ui targeted styles into the above block which is also .toc ?

Reply:
💥 25df2b8

Reply:
woot woot!! 😀